samples/apps/copilot-chat-app/webapi/CopilotChat/Skills/SemanticSkills/Chat/config.json

```diff
@@ -0,0 +1,81 @@
+{
+  "schema": 1,
+  "description": "Generate a chat response from the underlying LLM",
+  "type": "completion",
+  "completion": {
+    "max_tokens": 2048,
+    "temperature": 0.7,
+    "top_p": 1.0,
+    "presence_penalty": 0.5,
+    "frequency_penalty": 0.5
+  },
+  "input": {
+    "parameters": [
```

Comment:
Does the prompt error out when all parameters that appear in the context aren't explicitly defined? If not, some of the parameters defined here aren't relevant to the prompt correlating to the config so they can be removed, i.e., prompt, chatId, proposedPlan

Also, it doesn't define some of the parameters that are expected in the prompt, i.e., Audience

Reply:
Prompts dont error out when a parameter is not explicitly defined.
Params like $Audience are added to the context programatically instead of through the config.json; I'll add in these params for clarity/readability but they are not needed.
The core kernel throws errors if it expects an input and finds no param for it. Theses arrors are already propagated up to the users.